.env.example

```
@@ -0,0 +1,5 @@
+PAYLOAD_SECRET=
+DATABASE_URL=file:./payload.db
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DATABASE_URL key should go before the PAYLOAD_SECRET key

(UnorderedKey)